src/pages/iou/IOUCurrencySelection.js

```diff
@@ -79,7 +79,7 @@ class IOUCurrencySelection extends Component {
     getCurrencyOptions() {
         return _.map(this.props.currencyList, (currencyInfo, currencyCode) => ({
             text: `${currencyCode} - ${CurrencySymbolUtils.getLocalizedCurrencySymbol(this.props.preferredLocale, currencyCode)}`,
-            searchText: `${currencyCode} ${currencyInfo.symbol}`,
+            searchText: `${currencyCode} - ${CurrencySymbolUtils.getLocalizedCurrencySymbol(this.props.preferredLocale, currencyCode)}`,
```

Comment:
Are we okay repeating it twice? 

Should we remove and define it once and use it for both text and SearchText? 
Or
Removing `searchText` for good use `text` instead ?

@puneetlath  What do you say?

Reply:
I'm with removing `searchText`

Reply:
@Santhosh-Sellavel 
right now `text` has `-` which may not be needed in `searchText` would there ever be a case where both won't match entirely?

Reply:
@zereraz please make the changes!

Reply:
Pushed the changes.